app/scripts/views/sign_in.js

```diff
@@ -84,6 +84,7 @@ define(function (require, exports, module) {
         isAmoMigration: this.isAmoMigration(),
         isSyncMigration: this.isSyncMigration(),
         password: this._formPrefill.get('password'),
+        queryString: this.getCurrentQueryString(),
```

Comment:
If we call them `Search` parameters elsewhere, should we call it a `searchString` here and in the templates?